saleor/graphql/checkout/tests/test_checkout.py

```diff
@@ -180,6 +180,7 @@ def test_checkout_create_with_default_channel(
         response = api_client.post_graphql(MUTATION_CHECKOUT_CREATE, variables)
         content = get_graphql_content(response)["data"]["checkoutCreate"]
 
+    # DEPRECATED
     assert content["created"] is True
```

Comment:
IMHO you could just remove the deprecated field from the tests. In that case, this field doesn't do anything, it just always returns True.